server.js

```diff
@@ -16,261 +16,246 @@ var polyfillio = require('polyfill-service');
 var PolyfillSet = require('./scripts/PolyfillSet.js');
```

Comment:
I think you'll need to move some/all of these variable declarations into the export function